fmriprep/cli/parser.py

```diff
@@ -55,11 +55,14 @@ def _filter_pybids_none_any(dct):
             for k, v in dct.items()
         }
 
-    def _bids_filter(value):
+    def _bids_filter(value, parser):
         from json import loads
 
-        if value and Path(value).exists():
-            return loads(Path(value).read_text(), object_hook=_filter_pybids_none_any)
+        if value:
+            if Path(value).exists():
+                return loads(Path(value).read_text(), object_hook=_filter_pybids_none_any)
+            else:
+                raise parser.error(f"Path does not exist: <{value}>.")
```

Comment:
Let's follow the lead of `_is_file` and delegate the basic check while adding a more informative error for bad JSON:

```suggestion
        path = _is_file(value, parser)
        try:
            return loads(path.read_text(), object_hook=_filter_pybids_none_any)
        except JSONDecodeError:
            parser.error(f"Invalid JSON file: <{value}>")
```